Improving docs (#2)
jemmaissroff/ruby: doc/contributing/making_changes_to_ruby.md

Most Ruby documentation lives in the source files and is written in [rdoc format](https://ruby.github.io/rdoc/RDoc/Markup.html).

Some guidelines live under the `doc` folder and can be written in either `rdoc` or `markdown` format, determined by the file extension.

> To generate the output of documentation changes in HTML in the `{build folder}/.ext/html` directory, run

Author: I feel that we should mention it's under the build folder (based on previous steps, it should be `ruby/build`). Otherwise it's confusing to beginners like me.

> To generate the output of documentation changes in HTML in the `{build folder}/.ext/html` directory, run
> 
> ```
> make html
> ```
> 
> under your build folder. Then you can preview your changes by opening `{build folder}/.ext/html/index.html` file in your browser.

Reviewer: I'm not a fan of a code block breaking a sentence:

````suggestion
To generate the output of documentation changes in HTML in the `{build folder}/.ext/html` directory, run the following inside your build directory:

```
make html
```

Then you can preview your changes by opening `{build folder}/.ext/html/index.html` file in your browser.
````

> under your build folder. Then you can preview your changes by opening `{build folder}/.ext/html/index.html` file in your browser.

Reviewer: I'm not a fan of a code block breaking up a sentence

````suggestion
To generate the output of documentation changes in HTML in the `{build folder}/.ext/html` directory, run the following command inside your build directory:

```
make html
```

Then you can preview your changes by opening `{build folder}/.ext/html/index.html` file in your browser.
